iOSNativeLoginTemplate/iOSNativeLoginTemplate/SceneDelegate.swift

```diff
@@ -129,3 +163,28 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
        postResetBlock()
    }
 }
+
+@MainActor class ReCaptchaClientObservable: ObservableObject {
```

Comment:
When Combine first came out in 2019, I remember creating lots of these higher-level objects myself.  It's so cool to see how it's grown up in SwiftUI's context 🎉